modules/openy_features/openy_demo_content/modules/openy_demo_nlanding/config/install/migrate_plus.migration.openy_demo_node_landing.yml

```diff
@@ -44,10 +44,10 @@ source:
       alias: '/give'
       layout: one_column
       header_content_ids:
-        -
-          header_content_id: give_microsites_menu
         -
           header_content_id: give_gallery
+        -
+          header_content_id: give_microsites_menu
```

Comment:
@podarok we've changed weight of microsites menu because at Carnation it should be displayed below banner.
For Lilly, Rose it is not so important, bc header is not transparent.